lesson-4-calculator/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Calculator, Lesson 4</title>
+  <link rel="stylesheet" href="main.css">
+</head>
+<body>
+  <section>
+    <div class="output">
+      <ul id="number-list">
+      </ul>
+      <p id="result">0</p>
+    </div>
+    <div class="form">
+      <input id="input-form" type="text"></input>  
+      <button id="sum-button"><span>Add</span></button>
+    </div>
+  </section>
+
+  <script type="text/javascript">
+    const inputEl = document.getElementById('input-form');
+    const btnEl = document.getElementById('sum-button');
+    const listEl = document.getElementById('number-list');
+    const resultEl = document.getElementById('result');
+    let sum = 0;
+
+    function addToListAndSum() {
+      const number = parseFloat(inputEl.value);
+      if (!number)
+        return;
```

Comment:
This can be written in one line:   if (!number) return;

Reply:
done